rotkehlchen/api/v1/resources.py

```diff
@@ -1908,6 +1908,25 @@ def get(self, async_query: bool) -> Response:
         return self.rest_api.get_aave_balances(async_query=async_query)
 
 
+class AaveStatsResource(BaseMethodView):
+
+    get_schema = HistoryProcessingSchema()
```

Comment:
Is missing the docs

Reply:
you mean api.rst docs? When you planning to add em? Why did we need this btw? Didn't we already have one endpoint for this before? We should try to keep number of endpoints and api calls contained.

Reply:
What I mostly meant here is we should think in the future to also reduce amount of api endpoints and calls. Perhaps no need to have one endpoint per stats per protocol. Instead make the protocol an argument etc.

Reply:
Okey got it. Will see in a different PR how we can handle this. Atm we have liquity and aave and the kraken staking. Need to think how to abstract them